content/_data/changelogs/weekly.yml

```diff
@@ -4745,6 +4745,70 @@
         Internal: Update slf4j version from 1.7.25 to 1.7.26
     # pull: 4097 internal: [WEBSITE-634] - Add Release Drafter configuration for drafting Weekly release YAMLs
     # pull: 4112 too minor (Fixing hint for Jenkins.getInstance)
+- version: '2.188'
+  date: 2019-08-07
+  banner: >
+    This release failed. No artifact or Git tag exists.
+  changes: []
+- version: '2.189'
+  date: 2019-08-07
+  changes:
+    - type: bug
+      pull: 4137
+      issue: 58733
+      message: |-
+        A file handle leak in <code>$JENKINS_HOME/jobs/*/builds/permalinks</code> could prevent jobs from being deleted on Windows. (regression in 2.185)
+    - type: bug
+      pull: 4133
+      issue: 58548
+      message: |-
+        Remove extra whitespace output from <code>/scriptText</code> endpoint. (regression in 2.186)
```

Comment:
```suggestion
        Remove extra whitespace output from <code>/scriptText</code> endpoint (regression in 2.186).
```

Reply:
There are _many_ occurrences of this format, would recommend changing all of them rather than just this one.